### PR TITLE
add Feature.SlackDigestNotification

### PR DESCRIPTION
**Pull request by camhashemi**
EDIT: don't listen to this, see below comment

this is my favorite of the three options I mentioned here: https://kifi.slack.com/archives/G0K0RKDKQ/p1453342261000025, because it fits in with the rest of our configurable permissions framework. 

team admins will choose who can edit slack integrations and their settings (cc @JRuff42, this is a new setting in /settings, the "who can create" doesn't make sense since it can be "anyone").

### CODE REVIEW

**Comment by camhashemi** [comment]
talked with Jen, change of plans: we'll use the existing `Feature.CREATE_SLACK_INTEGRATION` to configure who can create / edit integrations within a team. Therefore non-members can't create integrations on team libraries.

**Comment by camhashemi** [comment]
@ryanpbrewster @andrewconner this is ready to go

**Comment by andrewconner** [comment]
If it does the right thing, :+1: :)

File: kifi-backend/modules/shoebox/marketing [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 7233683e26ec6ea2a44cd5ead340994e10da771a
+Subproject commit 25c419cc54cf6ee9f724fea995a2affbfb0e06e9
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/marketing:1]: intended?

**Comment by ryanpbrewster** [comment]
Sweet, looks nice. I like the comments and tests.

I'll need to take another look over `editableWith`, I'm not sure I really understand that yet.

File: kifi-backend/modules/shoebox/marketing (hunk 1 above)
Comment (camhashemi) [inline kifi-backend/modules/shoebox/marketing:1]: not intended, but up-to-date

**Comment by camhashemi** [comment]
e.g. sometimes you need `OrganizationPermission.MANAGE_PLAN` to edit a setting (was the default until now), sometimes you need `OrganizationPermission.CREATE_SLACK_INTEGRATION` to edit a setting. `Feature.editableWith` is the permission you need to edit settings for that `Feature`.

**Pull request merged**